### PR TITLE
docs: clarify Alpine install path in install.md

### PR DESCRIPTION
**Pull request by bpmct**
From install.md, the path to install code-server on Linux Alpine is unclear.

#1430 recommends using the npm script, since the install script doesn't work.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -419,7 +419,7 @@ install_npm() {
   echoh
   echoerr "Please install npm or yarn to install code-server!"
   echoerr "You will need at least node v12 and a few C dependencies."
-  echoerr "See the docs https://github.com/cdr/code-server#yarn-npm"
+  echoerr "See the docs https://github.com/cdr/code-server/blob/master/docs/install.md#yarn-npm"
```
Comment (jsjoeio) [inline install.sh:422]: This should probably point to the version branch, right? That way it always matches the latest released version of code-server. 

(And when we update release, we update links like these so they're always up-to-date!)
```suggestion
  echoerr "See the docs https://github.com/cdr/code-server/blob/v3.8.1/docs/install.md#yarn-npm"
```

**Review by jsjoeio — CHANGES_REQUESTED** [review]
Looks great! Thanks for adding this, Ben! 🙌

One small change

File: install.sh (hunk 1 above)
Comment (bpmct) [reply install.sh:422]: I noticed two other links in this file reference the master doc. Should we make all of them reflect the current version? Also, would it be possible to use the `$VERSION` var in this so we don't have to manually change them?

Or are the links auto-updated with some job?

Comment (jsjoeio) [reply install.sh:422]: Oh good catch!

> Should we make all of them reflect the current version

Yes!

> Also, would it be possible to use the $VERSION var in this so we don't have to manually change them?

I think so? But I'm not sure. I wasn't sure if that was defined globally or on a script-by-script basis. I'll defer to you though - whichever option you think is best. 

> Or are the links auto-updated with some job?

Right now it's manual with a "Find and Replace all" as part of the release process, but I do want to automate it.

**Review by jsjoeio — APPROVED** [review]
image

**Comment by jsjoeio** [comment]
@bpmct just realized - can you sign your commits so that they're verified and we can merge this? 

If not, happy to pull down, rebase and sign them for you!

**Comment by bpmct** [comment]
@jsjoeio done, sorry about that!

**Pull request merged**